### PR TITLE
fix deprecation warning for django 1.8 with importlib

### PR DESCRIPTION
**Pull request by kazmiruk**
It hides irritating warnings in django 1.8

### CODE REVIEW

File: django_jenkins/management/commands/jenkins.py [hunk 1]
```diff
@@ -194,7 +198,10 @@ def get_tested_locations(self, test_labels):
                     warnings.warn('No app found for test: {0}'.format(test_label))
         except ImportError:
             # django 1.6
-            from django.utils.importlib import import_module
```
Comment (kmmbvnr) [inline django_jenkins/management/commands/jenkins.py]: This one unnecessary.

**Comment by kmmbvnr** [comment]
importlib should be in install_reqs as here - https://github.com/kmmbvnr/django-jenkins/blob/975e8da206f7ed495302fe27b0951e13e575b966/setup.py#L16

**Comment by kazmiruk** [comment]
Done with install_reqs

File: setup.py [hunk 2]
```diff
@@ -8,6 +8,15 @@
 
 read = lambda filepath: codecs.open(filepath, 'r', 'utf-8').read()
 
+install_requires = [
+    'Django>=1.6',
+]
+
+# Needed for Python <2.7
```
Comment (nikolas2) [inline setup.py]: django-jenkins isn't being tested on Python 2.6, so it doesn't officially support Python 2.6.

This should either be removed, or you should add a new python2.6 tox env to this PR so we can make sure it works.

Comment (kmmbvnr) [inline setup.py]: Actually it have python2.6 support. And yep tox.ini entries are missed

**Comment by JshWright** [comment]
What's pending to get this merged?

**Comment by michaelhelmick** [comment]
Also looking for this to be merged +1

**Comment by laz2** [comment]
+1

**Comment by Pasql** [comment]
+1

**Comment by dhirajt** [comment]
@kmmbvnr Please merge this ASAP and release on pypi. Thanks!

**Comment by JshWright** [comment]
As the first Django 1.9 alpha has been cut, it there any movement on this PR? From the comments, it is not clear if any further work is needed. If there is work to be done, please outline it and I will be happy to polish it up.

**Comment by kazmiruk** [comment]
I could finish it by myself, but I don't understand what is wrong in this PR.

**Comment by michaelhelmick** [comment]
@kmmbvnr Hate to sound like a pest, but any idea when this could be merged? Looks like the changes that were asked were made? Like @JshWright said, with Django 1.9a being released, this will need to be merged to be compatible with Django 1.9 being that it will be removed.

```
RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
from django.utils.importlib import import_module
```

**Comment by kmmbvnr** [comment]
Sorry for the late release, I hope single warning on jenkins was not hurt you much for your big django projects, but anyway, nowadays django 1.6 support could be dropped and this patch is unnecessary.

**Pull request closed without merging**